plugin/trino-starrocks/src/main/java/io/trino/plugin/starrocks/StarRocksClient.java

```diff
@@ -0,0 +1,661 @@
+/*
```

Comment:
Since it supports MySQL protocol can we use `MysqlClient` here ?

Reply:
Some mysql specify properties on office doc did not make clear whether they support or not. StarRocks also has some specified properties and DDL statement, from the system evolve persective I think we may not use`MysqlClient` directly, and Using `MysqlClient` also need depend on mysql plugin it's a little bit overkill.